pyface/tasks/i_dock_pane.py

```diff
@@ -36,6 +36,13 @@ class IDockPane(ITaskPane):
     # Whether the pane is currently visible.
     visible = Bool(False)
 
+    # Whether the title bar of the pane is currently visible.
+    caption_visible = Bool(True)
+
+    # AUI ring number; note that panes won't be movable out of their ring
+    # number.  This is a way to isolate panes
+    dock_layer = Int(0)
```

Comment:
I think that this Trait should be on the Wx `DockPane` implementation, since its specific to AUI.